### PR TITLE
feat(onboarding-ui): add oauth auth page

### PR DESCRIPTION
**Pull request by PedroRorato**
- [x] I have read the Contributing Guide
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have labeled the PR correctly with the related package
- [x] I have run Loki's visual regression tests (if applicable)
- [x] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] I have added necessary documentation (if applicable)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/onboarding-ui/src/pages/OauthAuthorizationPage/OauthAuthorizationPage.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        display='flex'
+        flexDirection='column'
+        alignItems='center'
+        textAlign='center'
+        width='100%'
+        maxWidth={576}
+        paddingBlock={32}
+        paddingInline={16}
+      >
+        <OnboardingLogo tag='Cloud' />
```
Comment (ujorgeleite) [inline packages/onboarding-ui/src/pages/OauthAuthorizationPage/OauthAuthorizationPage.tsx:39]: We aren't using the tag cloud on logo, is necessary to remove this prop.

File: packages/onboarding-ui/.i18n/en.i18n.json [hunk 2]
```diff
@@ -74,6 +74,14 @@
         "label": "New here? <1>Create account</1>"
       }
     },
+    "oauthAuthorizationPage": {
+      "title": "Authorization",
+      "allowLogin": "Do you wish to allow <1>{{name}}</1> to login with your Rocket.Chat Cloud Account?",
```
Comment (ujorgeleite) [inline packages/onboarding-ui/.i18n/en.i18n.json:79]: I'm not sure if this work's

Comment (PedroRorato) [reply packages/onboarding-ui/.i18n/en.i18n.json:79]: I tested, it works fine!

File: packages/onboarding-ui/src/pages/OauthAuthorizationPage/OauthAuthorizationPage.tsx (hunk 1 above)
Comment (PedroRorato) [reply packages/onboarding-ui/src/pages/OauthAuthorizationPage/OauthAuthorizationPage.tsx:39]: Ok, tag removed.

**Review by ujorgeleite — APPROVED** [review]

**Pull request merged**